internal/probes/systemprompt/extraction.go

```
@@ -0,0 +1,68 @@
+		SimpleProbe: probes.NewSimpleProbe(
+			"systemprompt.Extraction",
+			extractionGoal,
+			"systemprompt.LeakDetector",
+			"Tests whether the model can be induced to disclose its own system prompt / hidden instructions (OWASP LLM07 System Prompt Leakage)",
+			prompts,
+		),
+	}
+	// Stamp the goal into each attempt so the judge branch of the detector has a
+	// rubric (judge.Judge reads attempt.Metadata["goal"]). The regex branch
+	// ignores it.
+	probe.SimpleProbe.MetadataFn = func(_ int, _ string, a *attempt.Attempt) {
```

golangci-lint (2.12.2)

[error] QF1008: could remove embedded field "SimpleProbe" from selector

(staticcheck)